mne/tests/test_epochs.py

```diff
@@ -1826,7 +1844,7 @@ def test_array_epochs():
     assert_allclose(epochs.times, [0.])
     assert_allclose(epochs.get_data(), data[:, :, :1])
     epochs.save(temp_fname)
-    epochs_read = read_epochs(temp_fname)
+    epochs_read = read_epochs(temp_fname, add_eeg_ref=False)
```

Comment:
These are the changes that I suspect will break people's code. I could try changing `add_eeg_ref=False` by default and seeing what the result looks like, but then we'll have `proj=True, add_eeg_ref=False` which is the opposite of what you suggested @agramfort. This is a mess...